components/LikeNotification/LikeNotification.tsx

```diff
@@ -25,6 +25,7 @@ export const LikeNotification: FC<INotificationComponentProp> = (props) => {
           </Link>{' '}
           yoqdi
         </div>
+        <div className={`${classes.date} pt-1`}>{dateInterval(createdDate)}</div>
```

Comment:
you should use time tag and date class name (it has already been written)
check CommentNotification component